src/redux/actions/userActions.js

```diff
@@ -10,6 +10,7 @@ export const POST_INBOX_SUCCESS = 'POST_INBOX_SUCCESS';
 export const SET_ACTIVE_CONVERSATION = 'SET_ACTIVE_CONVERSATION';
 
 export const getCurrentUser = (idToken, oktaAuth) => async dispatch => {
+  console.log(oktaAuth);
```

Comment:
is this necessary?

Reply:
this isn't code we implemented their was a pr earlier last week fixing the logout button and some auth stuff that we copy pasted over to get the website running on our end I'm assuming the console logs are their because before their was a console.log in the navbar revealing the role_id

Reply:
I just checked to see if the user ended up removing them which he did so we'll remove them